docs/administration/configuring-yoda.md

```diff
@@ -163,6 +163,7 @@ yoda_portal_wsgi_daemon_threads   | The number of threads to be created to handl
 yoda_portal_upload_part_files     | Whether the portal uploader function should upload multi-chunk files as .part files initially and rename them to their final name later (boolean value, default: true). It is generally recommended to keep this enabled, so that users can easily see when an upload failed and the result is partial. However, on storage systems where renaming data objects takes much time, such as S3 object storage in consistent mode, it may be necessary to switch use of .part files off.
 yoda_portal_monitor_signal_file   | Path to the signal file for the portal monitoring thread. If this file is present, the monitor thread will start logging technical support information to the directory configured in `yoda_portal_monitor_output_dir`. Set to empty string to disable the monitoring thread. Default value: /var/www/yoda/show-tech.sig
 yoda_portal_monitor_output_dir    | Path for the portal monitoring thread to log technical support information to if the signal file is present. Default value: /tmp. Note that the portal uses a private /tmp directory, rather than the global one.
+anubis_enable                     | Whether to enable Anubis, the web scraper blocker (currently only protects anonymous webdav)
```

Comment:
Can we change this to enable_anubis? To make it similar to the other enable_* variables.